src/pyj/read_book/flow_mode.pyj

```diff
@@ -10,6 +10,9 @@ from read_book.viewport import scroll_viewport
 from utils import document_height, viewport_to_document
 
 
+def line_height():
+    return parseFloat(line_height.doc_style.lineHeight)
```

Comment:
are u sure computed style line height is always a pixel value on all browsers?

Reply:
It should be if they follow spec according to this:
- https://developer.mozilla.org/en-US/docs/Web/CSS/computed_value
- https://drafts.csswg.org/css2/visudet.html#propdef-line-height

> The computation needed to reach a property's computed value typically involves converting relative values (such as those in em units or percentages) to absolute values. For example, if an element has specified values font-size: 16px and padding-top: 2em, then the computed value of padding-top is 32px (double the font size).

After reading the spec though, there is a special case value "normal" that will be shown as that, in which they recommend using 1 to 1.2 * fontSize. So I will add a fallback for that case.